railties/lib/rails/generators/rails/app/templates/config/boot.rb.tt

```diff
@@ -1,5 +1,10 @@
 ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", __dir__)
 
+# Enable frozen string literals
```

Comment:
We probably need some long comment explaining to remove this if you run into issues in dependencies, etc.